heroku-bouncer.gemspec

```diff
@@ -25,20 +25,20 @@ Gem::Specification.new do |s|
     "Rakefile",
   ])
   s.license = "MIT"
-  s.required_ruby_version = ">= 2.2"
+  s.required_ruby_version = ">= 3.1"
 
   s.add_runtime_dependency("omniauth-heroku", [">= 0.1", "< 2"])
-  s.add_runtime_dependency("sinatra", ">= 1.0", "< 3")
+  s.add_runtime_dependency("sinatra", ">= 3.0", "< 4")
   s.add_runtime_dependency("faraday", ">= 0.8", "< 2")
-  s.add_runtime_dependency("rack", ">= 1.0", "< 3")
+  s.add_runtime_dependency("rack", ">= 2.0", "< 3")
```

Comment:
Current rack is 3.0 we probably want to make the upper limit here 34.0

Reply:
Updated to `add_runtime_dependency("rack", ">= 2.0", "< 4")`, does that look right?